.github/workflows/chart-release.yml

```diff
@@ -0,0 +1,32 @@
+name: Chart Release
+on:
+  release:
+    types: [published]
+
+jobs:
+  release:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+    
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
```

Comment:
please check if this can be removed, AFAIK, the checkout step does this automatically.